### DISCUSSION TITLE
Argo-cd support for arm64 based docker images

### DISCUSSION THREAD

**Discussion by manee2k6** [argoproj/argo-cd#6830] [Ideas]
Hi all,

  I am very keen to understand that does argo-cd has support for arm64 based digest in both docker hub and quay. As i notice it currently supports only amd64.

  Please help me support multi digest images in the registry, as we are completely into graviton 2 based instances in our k8s cluster.

**Comment by jopit** [comment]
There's an open feature request for this: #4211
